MicroPython/src/communication/API_communication.py

```diff
@@ -26,7 +26,7 @@ def authorization_request() -> str:
         logging.info("Failed to authorize in API {}".format(e))
         return None
 
-    if response.status_code != '200':
+    if response.status_code != '200': #and response.status_code != 200:
```

Comment:
I do not understand the comment. Additionally please use two spaces before comments and one after the hash e.g.:
if response.status_code != '200':  # my comment